workspace/utilities/lib/element.xsl

```diff
@@ -17,24 +19,29 @@
 			<xsl:if test="string-length($content) != 0">
 				<xsl:text>yes</xsl:text>
 			</xsl:if>
-			<!-- 
-				TODO: @Paco
-				Continue to check for case were we have only img or
-				other case without direct text 
-			-->
+			<xsl:if test="exslt:object-type($content) = 'node-set'">
+				<xsl:if test="count($content/*) != 0">
+					<xsl:text>yes</xsl:text>
+				</xsl:if>
+			</xsl:if>
+			<xsl:if test="count(exslt:node-set($content)/*) != 0">
+				<xsl:text>yes</xsl:text>
+			</xsl:if>
```

Comment:
multiple yes is not a problem. I juste use the yes word to create a string. any character in the has-text is checked with a string-length() != 0